torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -543,6 +541,15 @@ void Unpickler::readGlobal(
     // Unpickle a tensor
     bool quantized = class_name == "_rebuild_qtensor";
     rebuildTensor(quantized);
+  } else if (module_name == "builtins" && class_name == "complex") {
```

Comment:
above I see some stuff about floatlists -- we don't need to handle complexlists because we don't define them in e.g. native_functions yet?

Reply:
it's also a little unclear to me if this is handling endianness -- do you know?

Reply:
The real and imaginary parts of the complex number are pickled using `pushIValue` which seems to take care of it?
```
void Pickler::pushDouble(double value) {
  push<PickleOpCode>(PickleOpCode::BINFLOAT);
  // Python pickle format is big endian, swap.
  push<double>(swapDouble(value));
}
```

```
double Unpickler::readFloat() {
  AT_ASSERT(sizeof(double) == 8);
  double big_endian = read<double>();
  double little_endian;

  // Pickle floats are big endian, so reverse the bytes
  auto big_endian_ptr = reinterpret_cast<const char*>(&big_endian);
  std::reverse_copy(
      big_endian_ptr,
      big_endian_ptr + sizeof(big_endian),
      reinterpret_cast<char*>(&little_endian));

  return little_endian;
}
```